doc/api/repl.markdown

```diff
@@ -64,6 +64,9 @@ the following values:
    returns the formatting (including coloring) to display. Defaults to
    `util.inspect`.
 
+ - `completer` - an optional function that is used for Tab autocompletion.
+   See [Readline Interface][] for an example of using this.
```

Comment:
@thefourtheye Well, I ended direct linking to the page, I searched  a bit more through the docs I found a way to do it. Compiled the docs and the links worked as expected.